### PR TITLE
Add propagate upstream trace flags to Serverless / Step Functions

### PR DESCRIPTION
**Pull request by agocs**
### What does this PR do? What is the motivation?

We added `propagateUpstreamTrace` to `serverless-plugin-datadog / step functions` and `--propagate-upstream-trace` to `datadog-ci / step functions`. This updates the docs for instrumenting AWS step functions.

n.b. wait until https://github.com/DataDog/serverless-plugin-datadog/pull/492 is merged

### Merge instructions

- [ ] Please merge after reviewing

### CODE REVIEW

File: content/en/serverless/step_functions/installation.md [hunk 1]
```diff
@@ -70,10 +72,17 @@ For developers using [Serverless Framework][4] to deploy serverless applications
...
+    --forwarder <FORWARDER_ARN> \
+    --env <ENVIRONMENT> \
+    --propagate-upstream-trace
+
    ```
-   - Replace `<STEP_FUNCTION_ARN>` with the ARN of your Step Function.
+   - Replace `<STEP_FUNCTION_ARN>` with the ARN of your Step Function. Repeat the `--step-function` flag for each Step Function you wish to instrument.
    - Replace `<FORWARDER_ARN>` with the ARN of your Datadog Lambda Forwarder, as noted previously.
+   - Replace `<ENVIRONMENT>` with the environment tag you would like to apply to your Step Functions.
+   - `--propagate-upstream-trace` is optional, and will update your Step Function definitions to inject Step Function context into any downstream Step Function or Lambda invocations.
```
Comment (estherk15) [inline content/en/serverless/step_functions/installation.md:85]:
```suggestion
   - `--propagate-upstream-trace` is optional, and updates your Step Function definitions to inject Step Function context into any downstream Step Function or Lambda invocations.
```

**Review by estherk15 — APPROVED** [review]
Left one small edit, let me know when this is good to merge!

**Comment by agocs** [comment]
Thanks @estherk15 ! Probably tomorrow, I think my east coast colleagues have gone home for the day.

**Comment by agocs** [comment]
@estherk15 OK, we're good to go.

**Pull request merged**